docker/modules/Dockerfile.faux1

```diff
@@ -83,6 +83,7 @@ RUN mv /usr/sbin/tcpdump /usr/bin/tcpdump
 
 RUN $AG update && $AG install wpasupplicant vim iproute2
 
-COPY docker/include/etc/wpasupplicant/ /etc/wpasupplicant/
+COPY docker/include/etc/wpasupplicant/ /tmp/wpasupplicant/
+COPY docker/include/etc/wpasupplicant/cert/ /tmp/cert/
```

Comment:
this seems redundant -- why do you need this if you're copying the entire directory already?

Reply:
Yeah, that was because of the way the .conf file was configured. Changed.